examples/static/.gitignore

```diff
@@ -0,0 +1,16 @@
+# Logs
```

Comment:
I think we can skip gitignore for examples. The gitignore in the project root should cover these

Reply:
Agreed, but this was just my attempt (maybe we can add it to the main examples directory) to exclude package-lock.json -- primarily to avoid dependabot wanting to update it. Perhaps we can just configure dependabot to ignore package-lock within examples?

Reply:
I moved it up one (and simplified it)